core/services/directrequestocr/drlistener.go

```diff
@@ -23,6 +25,26 @@ import (
 var (
 	_ log.Listener   = &DRListener{}
 	_ job.ServiceCtx = &DRListener{}
+
+	promOracleEvents = promauto.NewCounterVec(prometheus.CounterOpts{
```

Comment:
What's the difference between separate counters and separate labels within a single counter? Can you still search/display by the label?

Reply:
A single counter will aggregate counters under single symbol. In Grafana you query a counter, then specify desired label values to get to actual values.